lib/utils.js

```diff
@@ -1,12 +1,13 @@
-const fs = require('fs')
-const jsonschema = require('jsonschema')
-
+import fs from 'node:fs'
+import { promisify } from 'node:util'
+import jsonschema from 'jsonschema'
+const readFile = promisify(fs.readFile)
```

Comment:
Sorry I missed this the first time round: you can just use https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fspromises_readfile_path_options (what you have is fine, just sharing in case you didn't know about this)